docs/components/index.md

```diff
@@ -190,6 +190,81 @@ If the value of the `ParentYear` property is changed by selecting the button in
 <p>Year: 1986</p>
 ```
 
+**Data binding methods with parameters**
```

Comment:
This isn't really about data binding. This is all event handling. Also, the event handler don't take additional parameters. But the code *inside* the event handler can contain whatever you like including code that closes over values in scope

We already mention at the end of the event handling section that you can use lambda syntax. I think it would be sufficient to expand on that statement to say that it's often convenient to close over additional value (like the current item in a `foreach` loop).